test/scripts/initialize_eks_cluster.bash

```diff
@@ -0,0 +1,48 @@
+LD_FLAGS=$1
+IMAGE_VERSION=$2
+EKS_CLUSTER_NAME=$3
+
+AWS_REGION="us-west-2"
+AWS_ACCOUNT_ID=$(aws --region $AWS_REGION sts get-caller-identity --query Account --output text)
+
+ECR_REGISTRY_NAME="csisecretdriver"
+ECR_REGISTRY_URL="$AWS_ACCOUNT_ID.dkr.ecr.$AWS_REGION.amazonaws.com"
+ECR_REGISTRY_URL_WITH_NAME="$ECR_REGISTRY_URL/$ECR_REGISTRY_NAME"
+
+ECR_IMAGE_NAME="csisecretdriver"
+NAMESPACE="kube-system"
+
+AWS_SERVICE_ACCOUNT_NAME="basic-test-mount-sa"
+
+docker build  --no-cache --build-arg LDFLAGS="$LD_FLAGS" -t $ECR_IMAGE_NAME -f ./docker/Dockerfile .
+aws --region $AWS_REGION ecr get-login-password | docker login --username AWS --password-stdin $ECR_REGISTRY_URL
+docker tag $ECR_IMAGE_NAME:latest $ECR_REGISTRY_URL_WITH_NAME:$IMAGE_VERSION
+docker push $ECR_REGISTRY_URL_WITH_NAME:$IMAGE_VERSION
+
+
+eksctl create cluster --name $EKS_CLUSTER_NAME --node-type m5.large --region $AWS_REGION
+
+eksctl utils associate-iam-oidc-provider --name $EKS_CLUSTER_NAME --approve --region $AWS_REGION
+
+eksctl create iamserviceaccount \
+     --name $AWS_SERVICE_ACCOUNT_NAME \
+     --namespace $NAMESPACE \
+     --cluster $EKS_CLUSTER_NAME \
+     --attach-policy-arn arn:aws:iam::aws:policy/AmazonSSMReadOnlyAccess \
+     --attach-policy-arn arn:aws:iam::aws:policy/SecretsManagerReadWrite \
+     --override-existing-serviceaccounts \
+     --approve \
+     --region $AWS_REGION
+
+helm install csi-secrets-store  manifest_staging/charts/secrets-store-csi-driver     --namespace $NAMESPACE \
```

Comment:
Could we just call `make e2e-helm-deploy` with the `REGISTRY=$ECR_REGISTRY_URL`? We should be able to reuse the `Makefile` targets for all of this.